staging/src/k8s.io/apiserver/pkg/server/options/authorization.go

```diff
@@ -187,5 +187,10 @@ func (s *DelegatingAuthorizationOptions) getClient() (kubernetes.Interface, erro
 	clientConfig.QPS = 200
 	clientConfig.Burst = 400
 
-	return kubernetes.NewForConfig(clientConfig)
+	// make the client use protobuf
+	protoConfig := rest.CopyConfig(clientConfig)
+	protoConfig.AcceptContentTypes = "application/vnd.kubernetes.protobuf,application/json"
+	protoConfig.ContentType = "application/vnd.kubernetes.protobuf"
+
+	return kubernetes.NewForConfig(protoConfig)
```

Comment:
this seems like a weird place to have to override this. Does this mean the webhook this points at must speak proto?

Reply:
This mechanism is explicitly about speaking to the "core" API server, i.e. kube-apiserver. Why should we not assume it speaks proto?

Reply:
@liggitt additional question... why we use `protoConfig.AcceptContentTypes` and negotiate to fallback to JSON, when we then force the proto anyway (general question, not just this case, but in other places in code).

Reply:
@liggitt why is this place weird? We load the kubeconfig some lines above. Where do you think we should decide about the wire format if not here? As @mfojtik writes, this returns a kube client. We should be able to assume that proto is the right format.

Reply:
You're right, I'm sorry... I thought this was the generic webhook setup. Does the same thing need to apply to the delegating authenticator?

Reply:
would scoping this to the `rest.InClusterConfig()` block be safer? there we're much more confident we're talking to the kube apiserver and not some arbitrary endpoint that speaks subjectaccessreviews

Reply:
last two questions are still relevant